src/app-mode-demo.js

```diff
@@ -498,4 +502,14 @@ document.addEventListener("DOMContentLoaded", () => {
 	};
 
 	esriGlobalNav.create({headerElm: '.esri-header-barrier', footerElm: '.esri-footer-barrier', menuData});
+
+	const header = document.querySelector('.esri-header-barrier');
+	header.addEventListener("header::title:save", (d) => {
+		const event = document.createEvent("CustomEvent");
+		event.initCustomEvent("header:update:inlineTitle", true, true, {
+			maxWidth: 380,
+			brandText: d.detail
+		});
+		document.querySelector(".esri-header-inline-title").dispatchEvent(event);
+	});
 });
```

Comment:
What's this all about? shouldn't  you just set the props in the data file?  (it's late maybe I'm just tired)

Reply:
Typically, yes. In this instance, I'm modifying the brand text after the person edits and submits it.